src/crypto-api/index.ts

```diff
@@ -330,14 +330,18 @@ export interface CryptoApi {
      * Bootstrap the secret storage by creating a new secret storage key, add it in the secret storage and
```

Comment:
What does this mean? Do we really create a new secret storage key and then add it to secret storage?

Reply:
ugh, no.

Reply:
I'm minded to simplify this in favour of the more detailed list.